<launch>
    <param name="/use_sim_time" value="true"/>
    <arg name="rvizconfig" default="$(find sdc)/rviz/base_gt.rviz"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

    <node pkg="map_server" type="map_server" name="map_server" args="/home/lab816/agv_ws/src/sdc/maps/20240705_test5_1map.yaml"/> 

    <node pkg="sdc" type="base_gt" name="base_gt" output="screen">
        <param name="do_calibration" value="true"/>
        <param name="base_link_initial_pose_x" value="0.0"/>
        <param name="base_link_initial_pose_y" value="0.0"/>
        <param name="base_link_initial_pose_yaw" value="0.0"/>
    </node>

</launch>  